internal/sandbox/sandbox.go

```diff
@@ -369,6 +402,11 @@ func (s *podmanSandbox) init() error {
 	} else {
 		return fmt.Errorf("error creating container: %w", err)
 	}
+	if len(s.copyArgs()) > 0 {
```

Comment:
Calling `s.copyArgs` makes the string slice twice.

Better to assign the result in the if condition. e.g:

```
if args := s.copyArgs(); len(args) > 0 {
  // etc
}
```

Reply:
Makes sense, made a change for this.